core/drivers/crypto/caam/hal/common/hal_cfg.c

```diff
@@ -52,12 +53,13 @@ enum caam_status caam_hal_cfg_get_conf(struct caam_jrcfg *jrcfg)
 		jrcfg->offset = (CFG_JR_INDEX + 1) * JRX_BLOCK_SIZE;
 		jrcfg->it_num = CFG_JR_INT;
 
-#ifdef CFG_NXP_CAAM_RUNTIME_JR
-		if (fdt) {
+		if (IS_ENABLED(CFG_NXP_CAAM_RUNTIME_JR) &&
+		    !is_embedded_dt(fdt)) {
+			if (fdt) {
 			/* Ensure Secure Job Ring is secure only into DTB */
-			caam_hal_cfg_disable_jobring_dt(fdt, jrcfg);
+				caam_hal_cfg_disable_jobring_dt(fdt, jrcfg);
+			}
```

Comment:
Fix indentation of the inline comment. Suggestion:
```c
		if (IS_ENABLED(CFG_NXP_CAAM_RUNTIME_JR) &&
		    fdt && !is_embedded_dt(fdt)) {
			/* Ensure Secure Job Ring is secure only into DTB */
			caam_hal_cfg_disable_jobring_dt(fdt, jrcfg);
		}

Reply:
Done